docs/notes/2.27.x.md

```diff
@@ -44,7 +44,7 @@ Minor fixes:
 
 ### Plugin API changes
 
-* Process execution can now specify a minimum amount of cpu cores required for a process to run via the `concurrency` field. `concurrency_available` is deprecated in favor of `concurrency`
+* Processes can now specify their `concurrency` requirements, influencing when Pants will execute them. Use `exclusive` to be the only running process, `exactly(n)` to require exactly `n` cpu cores, or `range(max=n, min=1)` to accept a value between `min` and `max` which is templated into the process's argv as `{pants_concurrency}`. The `concurrency_available` field is deprecated in favor of `concurrency`.
```

Comment:
The field is not officially deprecated (using our deprecation mechanism) yet, so best to say that this supersedes the `concurrency_available` field, which will be deprecated in the future. 

Then the PR that actually does the deprecation can overwrite this if it's in the same release...

Reply:
done